salt/utils/sanitisers.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
```

Comment:
So I should rename this to "sanitizers.py" or something? There is no such word, BTW, and my spelling gives an error... :cry: